src/screens/Profile/Home/index.tsx

```diff
@@ -169,29 +169,7 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
               <Sections.LinkItem
                 text={t(ProfileTexts.sections.account.linkItems.logout.label)}
                 icon={<ThemeIcon svg={LogOut} />}
-                onPress={() =>
-                  destructiveAlert({
-                    alertTitleString: t(
-                      ProfileTexts.sections.account.linkItems.logout
-                        .confirmTitle,
-                    ),
-                    alertMessageString: t(
-                      ProfileTexts.sections.account.linkItems.logout
-                        .confirmMessage,
-                    ),
-                    cancelAlertString: t(
-                      ProfileTexts.sections.account.linkItems.logout.alert
-                        .cancel,
-                    ),
-                    confirmAlertString: t(
-                      ProfileTexts.sections.account.linkItems.logout.alert
-                        .confirm,
-                    ),
-                    destructiveArrowFunction: () => {
-                      navigation.navigate('ConsequencesFromLogout');
-                    },
-                  })
-                }
+                onPress={() => navigation.navigate('ConsequencesFromLogout')}
```

Comment:
How will this be when ticketing is featured toggled out?

Reply:
I have fixed this already https://github.com/AtB-AS/mittatb-app/pull/2825/commits/09435fcc40d15454c1b86ec0f5c281925c8a54f8#diff-ad7eb34b5f49ab9a495ab3088652f0b4dfcfabcbc9981909c1c431b3afd48b65R189
Is this some old changeset? 🤔

Reply:
Ah, yes, I was going commit by commit. But it still removes confirmation but looking more into it the existing confirmation is more about ticketing also, so just not having it when disabling ticketing is probably fine 👍